app/tests/unit/test_web.py

```diff
@@ -24,7 +24,9 @@
 class TestBase(TestCase):
 
     test_config = dict(
-        TESTING=True, SQLALCHEMY_DATABASE_URI="sqlite:///:memory:"
+        TESTING=True,
+        SQLALCHEMY_DATABASE_URI="sqlite:///:memory:",
+        CACHE_TYPE="SimpleCache"
```

Comment:
why testing SimpleCache if you are introducing redis?

Reply:
Because fakeradis cannot simulate the use of flask caching, for django caching yes, you can run real radis, but this will drag along too many dependencies.